man/systemd.network.xml

```diff
@@ -706,6 +706,12 @@
             <citerefentry project='man-pages'><refentrytitle>inet_pton</refentrytitle><manvolnum>3</manvolnum></citerefentry>.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>OnDevice=</varname></term>
+          <listitem>
+            <para>A boolean. Specifies whether to take device as default route.</para>
```

Comment:
This should say what you just commented in the PR. "A boolean. Specifies whether the route shall be bound to this device or be bound to no specific device. Defaults to 'no'." or something like that.

Hmm, what is actually the default for this? Is it "false"? if so, shouldn't it be "true"? I mean, if the routes are configured in a .network file (i.e. something that is inherently bound to a specific interface), then I think the route should be bound to that interface by default too, unless the user explicitly excepts a route from that. Or does turning on "OnDevice=" have any negative effects that make this a bad choice?